explainers/hidden-content-explainer.md

```diff
@@ -135,56 +128,45 @@ sections already has an event handler to expand and show the section. With the
-be available. In particular, the page can know which section of text was found
-using find-in-page, fragment navigation, and scroll-to-text navigation.
-
-The developer may also be able to find out some information about what the user
-typed into a find-in-page dialog based on which elements receive an event.
-Likewise, for scroll-to-text navigation, the developer may be able to learn more
-about the content of the fragment search terms based on which section of the
-page receives an event.
-
-We believe that the benefit of providing this information to the site outweighs
-the potential risks. Moreover, the lower granularity information can already be
-approximated from scroll position and intersection observerations on the page.
-
-There are three cases to consider, each of which correspond to an action that
-caused the `beforematch` event to be fired:
-1. Find-in-page. The data typed into the search dialog for find-in-page is not
-   directly observable by the page. However, based on the `beforematch` event, the page
-   can infer that the searched text is within a particular element that received
-   the event. We believe that the risk of exposing this information to the page
-   is low:
-     * The granularity of information provided is low, since the event is
-       received by the containing element, and does not reveal the actual text
-       searched.
-     * Find-in-page shortcuts (such as CTRL-F) can already be intercepted by the
-       page, giving it opportunity to intercept user-typed "find-in-page"
-       requests and provide custom behavior.
-     * The position of the searched text can already be deduced by the page
-       based on scroll offsets and intersection observations.
-2. Scroll to text navigation. Depending on the implementation of scroll-to-text,
-   from the page's perspective the behavior is indistinguishable from
-   find-in-page behavior. This means that it reveals where the searched text is
-   located but not what it is. Furthemore, this information is only available to
-   the page using the `beforematch` event, and is not observable by any third
-   parties.
-
-Some consideration needs to be given to the `beforematch` event when it comes to
-cross frame interactions. Specifically, if the matched information is found within
-a frame, should the signal propagate to the embedding page?
-
-In general, we believe that the `beforematch` event does not pose a privacy problem.
-However, the privacy aspect of the event should be discussed in more detail via
-formal security reviews.
+The beforematch event could expose more information to the page than is
+currently exposed because it very directly hints to the text the user is
+searching for since it gets fired on the element containing the target text.
+
+Note that it is already possible to snoop on find-in-page by creating a very
+small scrollable area containing every next character the user could type into
```

Comment:
s/a very small/0 height/ 

Or I would just say this is separate movements: 
- It is possible to snoop this by creating a scrollable area containing every next ...
- Furthermore, it is possible to make this area very small (0px or 1px tall) so that the searched text is not visible to the user

Reply:
I reorganized it sort of like you suggested, how does it look?

Reply:
Looks good.